### ISSUE TITLE
Table shown in Database, but not in Table editor

### ISSUE THREAD

**Issue by peachp** [supabase/supabase#468]
# Bug report

## Describe the bug
I had a table `email_history`, which I believe I've deleted. It is no longer shown in "Table editor", but it is still shown under Database > Tables

## To Reproduce
Not sure how to reproduce. See above how it happened.

![image](https://user-images.githubusercontent.com/47209189/103543049-c78e0880-4e95-11eb-995c-ce2467f6f7da.png)

![image](https://user-images.githubusercontent.com/47209189/103543097-e096b980-4e95-11eb-94ee-89bdc8d6a06a.png)

**Comment by inian** [comment]
@peachp do you see the deleted table when you reload the page as well?

**Comment by peachp** [comment]
Yes, even when reloading with Ctrl+F5.
I can try delete it again from Database, but I thought I'll keep it for now for you to investigate why it is shown in one place but not another.

**Comment by inian** [comment]
thanks Waldemar! we will investigate and get back to you.

**Comment by inian** [comment]
Hi Waldemar, this issue has been fixed now. Thank you for reporting.

**Comment by peachp** [comment]
The table `email_history` is still showing under Database, but not under Table editor for me.
But I assume you fixed that it shouldn't happen in future, so I just deleted it from Database this time.

**Comment by kiwicopple** [comment]
@soedirgo, I wonder if this is related to pg-api returning some deleted tables?


> I had a table email_history, which I believe I've deleted.

@peachp how did you delete the table? Using the table editor or SQL?

**Comment by peachp** [comment]
@kiwicopple I don't think its possible to delete a table under Table editor. I deleted it under Database. Don't remember running SQL for that.

**Comment by soedirgo** [comment]
@peachp perhaps you mean the tab for `email_history` is missing? If click the table icon in the table editor image above, you can see all the tables.

It's a bit subtle, but there is a context menu when you right-click a tab in the table editor from which you can delete the table.

**Comment by peachp** [comment]
@soedirgo aaaah! That must be it. I must have closed it.

I see what you do here guys and why I got confused.
![image](https://user-images.githubusercontent.com/47209189/104093007-993a6f80-527f-11eb-9dcb-ca9bd4d544b2.png)

It's similar to Airtable: the button on the far left is to choose which tables to look at / edit, and the plus button on the right hand side after last tab is to create a new table. I got confused because Airtable (which I use a lot) **always** shows **all** tables. The button on the left hand side is only to jump between tables quickly.

For Supabase it makes sense to not **always** show **all** tables, because the projects built on Supabase are more likely to have many more tables. In Airtable we have over 30 tables, which already don't fit on my 32" screen, so I have to scroll horizontally... which is annoying, because I usually only work with like 4-7 tables at the same time withing periods of months.

However!
I would not compare Supabase to Airtable. Here is what I would do if I was in charge of UX here:
- Keep showing only a few selected tables at a time
- Move creating tables or any kind of DDL from **Table editor** away to **Database**
- Change the button on the right hand side after the last table tab to open existing tables

In other words:
Table editor: DML
Database: DDL

This is naturally subjective from my point of view and might not fit into your overall vision / UX design.

**Comment by kiwicopple** [comment]
Ah yeah I can see how that's confusing. 

@MildTomato @joshenlim let's add this one to the UX backlog.

Thanks for your patience on this one @peachp . I'll close the issue for now but feel free to continue commenting
